annotate.py

```diff
@@ -102,7 +115,13 @@ def annotate_img(img_path, initials, size1=1, size2=1) :
                     coords[1][i]="N"  
             break
 
-        
+        if(k==ord('c')):
```

Comment:
Why provide this (redundant) functionality, if left vs middle click already does this?

Reply:
Yes I agree it's redundant, I'll simplify the code and use left/middle click instead

Reply:
Fixed in new version of PR